content/en/docs/reference/_index.md

```diff
@@ -38,13 +38,15 @@ client libraries:
     * [JSONPath](/docs/reference/kubectl/jsonpath/) - Syntax guide for using [JSONPath expressions](http://goessner.net/articles/JsonPath/) with kubectl.
 * [kubeadm](/docs/reference/setup-tools/kubeadm/kubeadm/) - CLI tool to easily provision a secure Kubernetes cluster.
 
-## Config Reference
+## Components Reference
 
 * [kubelet](/docs/reference/command-line-tools-reference/kubelet/) - The primary *node agent* that runs on each node. The kubelet takes a set of PodSpecs and ensures that the described containers are running and healthy.
 * [kube-apiserver](/docs/reference/command-line-tools-reference/kube-apiserver/) - REST API that validates and configures data for API objects such as  pods, services, replication controllers.
 * [kube-controller-manager](/docs/reference/command-line-tools-reference/kube-controller-manager/) - Daemon that embeds the core control loops shipped with Kubernetes.
 * [kube-proxy](/docs/reference/command-line-tools-reference/kube-proxy/) - Can do simple TCP/UDP stream forwarding or round-robin TCP/UDP forwarding across a set of back-ends.
 * [kube-scheduler](/docs/reference/command-line-tools-reference/kube-scheduler/) - Scheduler that manages availability, performance, and capacity.
+  * [kube-scheduler Policies](/docs/reference/kube-scheduler/policies)
```

Comment:
@alculquicondor , I have not read through this page yet.
Is there a reason why these new links are included (and appear _under_ the component link)? The other links point to the auto-generated component pages.
It seems as if you are trying to group all scheduling (kube-scheduler) documentation into a single reference section?
I have not checked what is left in the concept section.

Reply:
Yes, that was the point. Do you have an alternative suggestion?